doc/releases/changelog-dev.md

```diff
@@ -238,6 +238,53 @@
   >>> H =  qml.ops.dot(coeffs, ops)
   ```
   
+* Added `pwc` as a convenience function for defining a `ParametrizedHamiltonian`.
+  This function can be used to create a callable coefficient by setting 
+  the timespan over which the function should be non-zero. The resulting callable 
+  can be passed an array of parameters and a time.
+  [(#3645)](https://github.com/PennyLaneAI/pennylane/pull/3645)
+
+  ```pycon
+  >>> timespan = (2, 4)
+  >>> f = pwc(timespan)
+  >>> f * qml.PauliX(0)
+  ParametrizedHamiltonian: terms=1
+  ``` 
+  The `params` array will be used as bin values evenly distributed over the timespan, 
+  and the parameter `t` will determine which of the bins is returned.
+
+  ```pycon
+  >>> f(params=[1.2, 2.3, 3.4, 4.5], t=3.9)
+  DeviceArray(4.5, dtype=float32) 
+  >>> f(params=[1.2, 2.3, 3.4, 4.5], t=6)  # zero outside the range (2, 4) 
+  DeviceArray(0., dtype=float32)
+  ```
+  
+* Added `pwc_from_function` as a decorator for defining a `ParametrizedHamiltonian`.
```

Comment:
Do we want this one, or just `pwc`?

Reply:
It would be awesome if we could have both functionalities under the same name.